src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Dto/MetaData.php

```diff
@@ -25,7 +25,6 @@ class MetaData
     /**
      * Supports 1 ACS location, the first entry is used that is passed from Manage
      */
-    private $acsBinding;
```

Comment:
Please explain why you are removing the acsBinding field from the metadata DTO. I see it is not used. But seeing this removed in this PR deserves at least a mention in the PR description or your commit message.

Reply:
Mentioned it in the commit message.